frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { Sighting } from "../../clients/apiClient";
+import "./BrowseSightings.scss";
+
+interface SightingProps {
+  sighting: Sighting;
+}
+
+export const SightingCard: React.FunctionComponent<SightingProps> = ({
+  sighting,
+}) => {
+  return (
+    <div className="sighting-card">
+      <h3 className="fade-in">{sighting.species.name}</h3>
+
+      {sighting.imageUrl !== "" ? (
+        <img
+          className="image fade-in"
+          src={sighting.imageUrl}
+          alt={sighting.species.name}
```

Comment:
As above, we can't guarantee having a species! Need to use something else for the alt-text